academy/golang-intro/1_install.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Go Introduction"
+order: 
+description: Get started with Golang
+tag: deep-dive
+---
+
+# Go Introduction
+
+Golang (Go) is an open-source, statically typed, and compiled programming language introduced by Google and first released in 2012. It has a BSD-style license. Its Git repository is located at [https://go.googlesource.com/go](https://go.googlesource.com/go).
+
+Start with the basics. In this section, you will install Go and learn some general things about it.
+
+
+## Install
+
+Installing Go depends on your machine. See the [Getting Started Page](https://golang.org/doc/install) and follow the instructions to install Go.
+
+<HighlightBox type="tip">
+
+[Gimme](https://github.com/travis-ci/gimme) is a useful script for Go.
```

Comment:
This does not look maintained anymore. I think people should stick with the official docs you've mentioned just above.